### PR TITLE
Update french content for portal

### PR DESCRIPTION
**Pull request by annemarieleger**
First pass of translation for all strings except "Home" part (lines 41-70)

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -1,90 +1,106 @@
...
   sign_in:
-    title: "COVID Shield Sign In"
-    error: "Invalid username or password"
-    username_placeholder: "Enter username"
-    password_placeholder: "Enter password"
-    sign_in: "Sign in"
+    title: "Se connecter à COVID Shield"
+    error: "Nom d'utilisateur ou mot de passe non valide"
+    username_placeholder: "Entrez le nom d'utilisateur"
+    password_placeholder: "Entrez le mot de passe "
```
Comment (emilybulger) [inline config/locales/fr.yml:38]: one extra space here before closing quote

File: config/locales/fr.yml [hunk 2]
```diff
@@ -1,90 +1,106 @@
...
+      title: "Fournir au patient un code COVID Shield"
+      subheading: "Dites au patient de:"
+      step1: "Appuyer sur l'icône pour ouvrir l'application COVID Shield."
+      step2: "Appuyer sur la section blanche au bas de l'écran."
+      step2_modal_title: "Étape 2"
+      step2_modal_content: "Appuyer sur la section blanche au bas de l'écran."
+      step2_modal_alt: "Capture d'écran de l'application mobile COVID Shield avec la section blanche au bas de l'écran encadrée"
+      step3_html: 'Appuyer sur le bouton <code class="inline-code">Entrez le code</code>.'
+      step3_modal_title: "Étape 3"
+      step3_modal_content_html: 'Appuyer sur le bouton <span class="inline-code">Entrez le code</span> .'
```
Comment (emilybulger) [inline config/locales/fr.yml:65]: space after </span> and before period

**Comment by dfmcphee** [comment]
I changed the copy for the user delete confirmation slightly in https://github.com/CovidShield/portal/pull/49/files#diff-e35182d90157bf50b09e78cb6a31defaL76

If we can get translations for those few strings I added I can rebase this and resolve the conflicts while I add them

File: config/locales/fr.yml (hunk 2 above)
Comment (annemarieleger) [reply config/locales/fr.yml]: There's only a space before the period. I removed it!

**Comment by annemarieleger** [comment]
@dfmcphee 
Sorry for the late reply. 

Here are the translated strings, not too sure where I should add them 😅 
```
    remove_user: "Supprimer un utilisateur"
    confirm_delete_html: "Vous êtes sur le point de supprimer <strong>%{user}</strong>."
    cannot_be_undone: "Cette action ne peut pas être annulée."
```

Let me know if I need to do anything.

**Pull request merged**